agent/wsclient/mock/utils/utils.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 // StartMockServer starts a mock websocket server.
-func StartMockServer(t *testing.T, closeWS <-chan bool) (*httptest.Server, chan<- string, <-chan string, <-chan error, error) {
+func StartMockServer(t *testing.T, closeWS <-chan []byte) (*httptest.Server, chan<- string, <-chan string, <-chan error, error) {
```

Comment:
You need to modify the copyright header for this file (`2015-2017`). Also please make sure that all updated code files have this updated.